File: ProjectTracker20cs/EnterpriseServicesHostcs/EnterpriseServicesProxy.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace EnterpriseServicesHost
{
  public class EnterpriseServicesProxy :
    Csla.DataPortalClient.EnterpriseServicesProxy
  {
    protected override 
      Csla.Server.Hosts.EnterpriseServicesPortal GetServerObject()
    {
      return new EnterpriseServicesPortal();
    }
  }
}

File: ProjectTrackervb/EnterpriseServicesHostvb/EnterpriseServicesProxy.vb
Public Class EnterpriseServicesProxy
  Inherits Csla.DataPortalClient.EnterpriseServicesProxy

  Protected Overrides Function GetServerObject() As _
    Csla.Server.Hosts.EnterpriseServicesPortal

    Return New EnterpriseServicesPortal

  End Function

End Class
